Dockerfile

```diff
@@ -45,17 +48,28 @@ RUN set -eux ; \
         file \
         curl-dev \
         libpng-dev \
+        jpeg-dev \ 
+        freetype-dev \
+	libwebp-dev \
```

Comment:
please format this line like other lines.

Reply:
Already formatted